config/config.go

```diff
@@ -60,6 +59,57 @@ type SessionsConfig struct {
 	MaxPerUser uint32
 }
 
+// Counter for the limits  and restrictions for the File Transfer
+//on and off the device(MEN-4325)
+type Counters struct {
+	// Maximum bytes count allowed to transfer per hours
+	// this is per device global limit, which is consulted
+	// every time there is a transfer starting. if above
+	// the limit, we answer with error message indicating
+	// limit reached.
+	MaxBytesTxPerHour uint64
+	MaxBytesRxPerHour uint64
```

Comment:
hmm. good question, those limits only work for the FileTransfer at the moment. I will think how to make them global. but if we ever make them global, then it makes better sense to update them async, right?

Reply:
ok I changed to per minute to match the exponentially weighted moving average that was introduced. about the global limit: I will create a followup task, once this one is in it can be easily extended. (I do not want to overcomplicate this PR) if thats acceptable?